TweetBane/app/src/main/java/com/elysium/tweetbane/MainActivity.java

```diff
@@ -0,0 +1,142 @@
+package com.elysium.tweetbane;
+
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+import java.io.IOException;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.FormBody;
+import okhttp3.Headers;
+import okhttp3.OkHttpClient;
+import okhttp3.Request;
+import okhttp3.RequestBody;
+import okhttp3.Response;
+
+public class MainActivity extends AppCompatActivity {
+
+    public static final String CONSUMER_KEY = "s004ntGGe3l3KoR9QnNFIA";
+    public static final String CONSUMER_SECRET = "C5bWYjYKIAE3e63Ps899uPMPhCnvbjqpQODTKWOQIc";
+
+
+    public static final String AccessToken = "27190812-CnEKQU7S5HJ1TVBOToQSbXxg1YFF93V7kE0pOFKIo";
```

Comment:
You should be doing your base64 encoding programmatically, rather than hard coding it, so that if your consumer key or consumer secret ever change you can just drop in the new values. Also, remember that constant names should be in all caps